client/components/button/test/index.js

```diff
@@ -0,0 +1,1064 @@
+/**
+ * Created by nitrajka on 09/10/16.
```

Comment:
Let's leave out comments like this, since it's better captured and maintained by git history.